src/component.tsx

```diff
@@ -19,13 +20,14 @@ const getStyles = (
   { left, top, right, gap }: GapOffset,
   allowRelative: boolean,
   gapMode: GapMode = 'margin',
-  important: string
+  important: string,
+  bodyClassName: string
 ) => `
   .${noScrollbarsClassName} {
    overflow: hidden ${important};
    padding-right: ${gap}px ${important};
   }
-  body {
+  body.${bodyClassName} {
```

Comment:
So something like
```tsx
body[data-scroll-locked] {
```

More importantly, this allows changing behaviour of this component like __statically defining all styles__ and operating only with data attribute.
This could be a good solution for #54